CHANGELOG.md

```diff
@@ -23,6 +23,7 @@
 ### Behavior Changes (API Compatible)
 
 - When parsing datatype during `to_pandas` operation, we rely on GS precision value to fix precision issue for large integer values. This may affect users where a column that was earlier returned as `int8` gets returned as `int64`. Users can fix this by explicitly specifying precision values for their return column.
+- `StoredProcedureRegistration` will now automatically add `snowflake-snowpark-python` as a package dependency. By default the lower of either the local version or the server version is used.
```

Comment:
Based on our discussion, we will pick the local client version and fail if server does not support it.